app/test_setup.go

```diff
@@ -61,6 +71,80 @@ func InitStrideTestApp(initChain bool) *StrideApp {
 	return app
 }
 
+func GenesisStateWithValSet(app *StrideApp) GenesisState {
```

Comment:
what was the reason for adding this? / were there some tests that broke until the full genesis state was mocked out?

Reply:
Since sdk 46, we can not init with an empty valset. That's why I use this function

Reply:
ah got it. Thanks for adding this!